<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="11" time="0.518" timestamp="2021-09-01T09:19:31.761384" hostname="WIN-0SHO48SNKUD"><testcase classname="tests.test_accum" name="test_accumulator_add_one" time="0.004" /><testcase classname="tests.test_accum" name="test_accumulator_add_three" time="0.004" /><testcase classname="tests.test_accum" name="test_accumulator_add_twice" time="0.006" /><testcase classname="tests.test_accum" name="test_accumulator_cannot_set_count_directly" time="0.005" /><testcase classname="tests.test_math" name="test_one_plus_one" time="0.005"><failure message="assert (1 + 1) == 0">def test_one_plus_one():
&gt;       assert 1+1 == 0
E       assert (1 + 1) == 0

tests\test_math.py:4: AssertionError</failure></testcase><testcase classname="tests.test_math" name="test_multiplication[2-3-6]" time="0.005" /><testcase classname="tests.test_math" name="test_multiplication[1-99-99]" time="0.015" /><testcase classname="tests.test_math" name="test_multiplication[0-99-0]" time="0.003" /><testcase classname="tests.test_math" name="test_multiplication[3--4--12]" time="0.013" /><testcase classname="tests.test_math" name="test_multiplication[-5--5-25]" time="0.005" /><testcase classname="tests.test_math" name="test_multiplication[2.5-6.7-16.75]" time="0.003" /></testsuite></testsuites>